templates/oauth_result.html.njk

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<meta charset="utf-8"/>
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+<link rel="shortcut icon" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAAAsSAAALEgHS3X78AAAG+ElEQVRo3u2aZ2hUWxSFY2+oUazYjVERVAQb9l4TDYoVRSxgrz8UO2JXrFgithTUkMSGvRcQQf0h9t57712XfBtOmAm+x3uZiW+G58Alk8y9566199pr73MnISEhIUp95MuXT6NHj9a1a9cUCC9wgAdcv8D7awJjxozR9evXA4IAOMDztwQyZMigjBkzKk+ePKpfv74WLVqku3fvBgQBcIAHXOADJ3i9CPBBrVq1NGDAAC1btkzHjx/Xq1evAoIAOMADLvCBE7xeBIoVK6ahQ4fqyJEjdsG3b9/0/fv3gCAADvCAC3zgBK8XgTJlymjmzJl6/vy518U/fvxIOX7n66/uCz5wgteLQNmyZTVr1qygIABO8HoRKF26tKZMmaLbt28HNAHwgRO8XgRKlSqlyZMn6+bNmz7dwJfz/8k54AMneP8QCGoCv7MO/hAIagIs8PXrV3369Env3r3Tixcv9PjxYz18+FBPnz613589e6Z79+7Z0HX16lX7id09ePDAfPvt27d27cuXL+1aZhvOuXz5sq5cuaJbt27p0aNH9vmHDx+s6/qtBgD/5s0bA3Pu3Dnt379fGzZsUFxcnDZv3qx9+/Zp586dWrt2rWbPnm0ePX36dC1cuFCxsbE6ePCgzp8/rwsXLujw4cNat26dlixZYt2Uc2fMmGEzTlJSko4ePWrE3r9/7z8Cnz9/tuicOXNG27Ztsy7Yt29fde/eXcOHDzcAXNezZ0/VrVtXlStXVrVq1dSwYUM7B6CbNm3Sli1bNH/+fPXr10+tW7e2gaxq1aqqUaOG2rZtq0GDBmnBggU6dOiQZdZvBD5+/Kg7d+7o2LFjWr16tfr37283r1Spkpo2bapevXrZ0bx5c1WsWFFFihRR/vz5VbBgQTunc+fOmjhxoqZNm6YhQ4YY2OrVq9sswzmFChVSWFiYkScwa9as0cWLF+2+TsI+EUCTN27cMCksXrzYAHFzNhfly5c3Eu3bt1dUVJQiIiLUoEEDhYeHK2/evHbwnmxERkaqU6dOdj3nNmrUyDJQrlw5myyLFy9uJMaNG6e9e/da0KgdJOwTAfTIlo5F586dayALFChgmwoiSDTbtWtnIy6aZuvXqlUrFS5cOGXTkS1bNlsf6YwYMcIyMnLkSPXo0cMCwFyTPXt2I9K1a1fLwsmTJ63uMA+fCeAsu3fvNj03a9ZMuXLlsmsASXTRNfrduHGjFTO/e0yLdpANMkUdrF+/3uSIrPr06WNSc0QhRKAOHDhgmUcBfsnAnj17rICbNGli0SK6VapU0cCBA7V06VKTGE6Dk7BOzZo1lTt3bnlOu4MHD9aOHTt09uxZnTp1Slu3bjUTIAiZM2dWpkyZVK9ePU2dOtXOw2ZTO1KaCGBt2OWcOXNSCJCFNm3apGw/nzx5Yo7F9cuXL7fPWBMSFHXt2rXtekCha9blfUJCgtUG52XJkkV16tTRpEmTzLUICP3D5yKGAP4PgMaNG1uq2ZN26dLF/JtG5F40o/j4eHXs2FEVKlQwl0HbSG/FihXW9NyL97t27TIXwxQgANEJEyaYHOk7FPJ/RsDZasmSJdWyZUutXLnSa8dH1pAKPcSTAEVO76ABpksGkBApBySdFSl8+fIlBRSFjFVisxQ6a0OADNAUPXdXRLpbt25W5FmzZvW/hFLXgCeBDh06WLRpPOifF7MODoO14vFIiAy0aNFC0dHRNkO5Jw24DBkkkxBIlxrw7AO4EI5BpChiosy8Q6qdXxPhVatWWcfFeZBG0aJFzR4peE8CBCYxMdGaGwHBiVwN+E1CnjZKH0hNICYmxorNtX4IoHVciMgTWbJA52WIozm5EeFXBBhTxo8f778i9swA0yYSypEjhxUxEnIZcASQEBmgYzNyYKFkAPulX7gMQMBJiO4bGhpq5oCE/FrEqWsAKeTMmdMiSxFTA2jVSQgCjAKuBhg3mHMY9ugPngS4V3JyshUxUqO2mIeoAUZ1v/UB/saYywNWIuu8nVmGRoQLuSLGhSAFKMZqJk1GBeRGcTsXggAuRLH27t1bJUqUsBkLouwntm/frkuXLvlOgMjev3/fhissc9iwYSYjbsTkyIxEH3A2yg4Nbx87dqxJgymUTI0aNcrAuj4AAbJBYLgvNkud8PAWoowkrOvzLAQwmhOLnThxwiLOQMaOC0DolI7qtoFIjlkHCVAL6B7/R+unT59OiSgEeFhL9tgokd158+ZZkNh7MECyrgtMmglgd8iDGxM95nT2sdyAzLx+/dqy5J5kM+fwNz5jLeqHg30wgfCc7wHntqusx7rICuC4D/dN/YQ83Z5K/NuHXGl9petjlaAm8L/LQFpJ/CEQsASC/huaoP+OLOi/pQz674mD/pv6oP9fiWD9b5WfSkWrKgYZxAsAAAAASUVORK5CYII=">
```

Comment:
uff.. this needs to be base64 inline, because we can't host files from the bridge?

IPFS fanboy note:
While possibly shitty because for the majority of users it depends on a centralised gateway, content-addressable IPFS-URLs may be an option.
https://gateway.pinata.cloud/ipfs/QmZwXKujr1UZPfVMqFS8ABLzJSaz8F2gE4Ky29JQthFGmw

Reply:
Hah, to be honest this was just to cut down on the number of moving parts. Serving files could be done, but felt like more effort than inlining some relatively small image files (ones SVG, the other is 48x48)

Reply:
We could use IPFS, but I just think it's the wrong place to introduce it. I really want this page to have zero dependencies and just work.